### DISCUSSION TITLE
Changeing layout of .eml -> .pdf for paperless-ngx

### DISCUSSION THREAD

**Discussion by viktorrauch** [gotenberg/gotenberg#857] [Q&A]
Hi community,

I set up paperless-ngx + apache-tika + gotenberg (docker on Synology).
Everything works fine. I put mails in a specific mailbox and paperless-ngx grabs them for archiving.

The only thing I want to change is the layout of the pdfs.
Currently the pdfs seem to be generated like this:

1. header with infos like sender, receiver, date etc.
2. clean text of the mail
3. html-version of the mail

I'd like to remove "2." completely or put it after "3.".
How exactly can I achieve this?

I sadly have no knowledge about scripting and very limited knowledge about docker. I already have portainer running if this helps.

**Comment by gulien** [comment]
Hello @viktorrauch,

I’m not familiar with paperless 😅 did you ask your question on their repository?
